modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -36,6 +54,20 @@ function h3ToPolygon(hexId) {
   return vertices;
 }
 
+function mergeTriggers(currentTrigger, newTrigger) {
+  let trigger;
+  if (currentTrigger === undefined || currentTrigger === null) {
+    trigger = newTrigger.coverage;
+  } else if (Array.isArray(currentTrigger)) {
+    trigger = [].concat(currentTrigger, newTrigger.coverage);
```

Comment:
`Object.assign({}, <array>)` works, you don't have to handle it as a special case.